src/include/ZividPython/DataModelWrapper.h

```diff
@@ -230,6 +254,9 @@ namespace ZividPython
                     pyClass.def(py::self < py::self); // NOLINT
                 }
 
+                pyClass.attr("is_array_like") = TypeIsZividRange<typename Target::ValueType>::value
```

Comment:
I rabbit-holed myself into attributes vs properties in python because I don't use attrs often. TIL while an attribute can be set by the user (`obj.is_array_like = False`), a read-only property can not be. Not asking to change it here, of course.